app/src/main/java/com/simplecity/amp_library/saf/SafManager.java

```
@@ -228,7 +227,6 @@ private String getExtSdCardFolder(final File file) {
                 }
             }
         } catch (NoSuchMethodError e) {
-            Crashlytics.log("getExtSdCardPaths() failed. " + e.getMessage());
```

Same here - no empty catch blocks